AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Index.cshtml

```diff
@@ -40,7 +40,7 @@
                         @if (!item.Published)
                         {
                             <div class="btn-group btn-group-sm row-btn-margin">
-                                <a data-toggle="tooltip" data-placement="top" title="Publish" asp-controller="Campaign" asp-action="Publish" asp-area="@AreaNames.Admin" asp-route-id="@item.Id" class="btn btn-primary"><span class="fa fa-upload"></span></a>
+                                <a data-toggle="tooltip" data-placement="top" title="Publish" asp-controller="Campaign" asp-action="Publish" asp-area="@AreaNames.Admin" asp-route-id="@item.Id" class="btn btn-primary"><span class="fa fa-upload"></span> Publish</a>
```

Comment:
I don't think the icon is helping anyone. Could you remove the <span class="fa fa-upload"></span> part so it's just the text label instead please?